stl/inc/xstring

```diff
@@ -2641,6 +2645,37 @@ private:
         }
     }
 
+#if _HAS_CXX20
+public:
+    // Move assign from a buffer, used by basic_stringbuf

```

Comment:
We conventionally place such comments inside functions, instead of outside. Occurs for the next function too.